src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Rultor-DynamoKey: ${dynamo.key}
 Rultor-DynamoSecret: ${dynamo.secret}
 Rultor-GithubId: ${github.id}
 Rultor-GithubSecret: ${github.secret}
+Rultor-GithubThreshold: ${github.threshold}
```

Comment:
Sure, I just wasn't sure what Rultor's convention was for what should be configurable vs. what to hardcode